R/add_r_files.R

```diff
@@ -83,3 +89,37 @@ add_utils <- function(
   )
 }
 
+#' Append roxygen comments to fct_ and utils_ files
+#' 
+#' This function as boilerplate roxygen comments 
+#' for fct_ and utils_ files
+#'
+#' @param name The name of the file
+#' @param path The path to the R script where the module will be written. 
+#' @param ext A string denoting the type of file to be created. 
+#' 
+#' @rdname file_creation
+#' @export
```

Comment:
Hey, 

I don't think this should be an exported function — chances are that there would be no reason to do `golem::append_roxygen_comment()` :) 

Can you replace this `@export` by a `@noRd`? 

Thanks :)

Reply:
No worries. New commit on its way.